Presentation.Web/Tests/Helpers/CatalogHelper.ts

```diff
@@ -0,0 +1,33 @@
+﻿import CatalogPage = require("../PageObjects/it-system/Catalog/ItSystemCatalog.po");
+import CSSLocator = require("../object-wrappers/CSSLocatorHelper");
+import Constants = require("../Utility/Constants");
+
+
+var consts = new Constants();
+
+class CatalogHelper {
+    public static createCatalog(name: string) {
+        var homePage = new CatalogPage();
+        var CSShelper = new CSSLocator();
+        homePage.getPage();
+        browser.wait(homePage.isLoginAvailable());
+        homePage.kendoToolbarWrapper.headerButtons().systemKatalogCreate.click();
+        element(CSShelper.byDataElementType(consts.nameOfSystemInput)).sendKeys(name);
```

Comment:
De to linjers detaljer gentages i deleteCatalog. Mangler der en wrapper/begreb omkring det?

Reply:
En metode som ikke er blevet lavet endnu som havde nogle placeholder kode, metoden er slettet da den ikke bruges.